tdd-list/list.js

```diff
@@ -6,7 +6,24 @@
  * @returns {array}
  */
 function paginate(data, page, limit) {
-    return [];
+  if (data === null || data === undefined) {
+    return null;
+  }
+
+  let startIndex = limit * page - limit;
+  let endIndex = limit * page;
+
+  if (startIndex > data.length) {
+    return null;
+  }
+
+  let users = data.slice(startIndex, endIndex);
+
+  if (users === undefined) {
```

Comment:
`users` nigdy nie będzie równe `undefined` - w przypadku gdy slice nie ma już co zwrócić, zwróci pustą tablicę i to powinniśmy zwrócić

Reply:
tak jest